11_FFTBloom/main.cpp

```diff
@@ -0,0 +1,1192 @@
+// Copyright (C) 2018-2024 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#include "nbl/application_templates/MonoDeviceApplication.hpp"
+#include "nbl/application_templates/MonoAssetManagerAndBuiltinResourceApplication.hpp"
+
+
+// TODO: Copy HelloSwapchain to set up a swapchain, draw to a surface. Or copy the new MPMC example.
+// TODO: Dynamically modify bloom size with mouse scroll/ +- keys
+//		 Notes: To do so must check required size against buffer size (expand buffer and recompile kernel if it starts being small), also after making kernel larger then small again
+//		        there's probably no need to shrink buffer and recompile kernel but would be nice to add that as well)
+// TODO: Refactor after FFT ext is back
+// TODO: Make sampling formats be #defined depending on how they were loaded on GPU side
```

Comment:
update the TODOs

Reply:
alos if you don't want to bother with handling input, just make the bloom size go up and down based on `cos(time)*cos(time)`